manifests/service.pp

```diff
@@ -7,10 +7,6 @@
   $service_manage = $chrony::service_manage,
   $service_name   = $chrony::service_name,
 ) inherits chrony {
```

Comment:
why does this class inherit from `chrony`
that's really bad form for anything other than the params class

Reply:
I have no idea. It has been that way since commit 74c0dc8 when the the file was added.

Reply:
okay
let's remove those in a future PR

i'm pretty, and sure, there's a standard puppet lint that should be complaining about this

have we gotten module/pdk sync yet for this module?